pipelines/live-1/main/check-environment.yaml

```diff
@@ -31,8 +31,14 @@ resource_types:
     repository: cfcommunity/slack-notification-resource
     tag: latest
 
+groups:
+- name: live-0
+  jobs: [Check-live-0]
+- name: live-1
+  jobs: [Check-live-1]
+
 jobs:
-- name: Check
+- name: Check-live-0
```

Comment:
One minor thing, can we please make this all in lowercase? Thanks! :)

Reply:
updated job names(check-live-0 and check-live-1) to all lowercase, considering the review comments.